Assets/VRTK/Scripts/VRTK_PlayerJump.cs

```diff
@@ -0,0 +1,366 @@
+            }
+
+            return scaled;
+        }
+
+        private void UpdateCrouchLowpoint()
+        {
+            if (HMDTransform.position.y < crouchLow)
+            {
+                crouchLow = HMDTransform.position.y;
+                crouchLowTime = Time.time;
+
+                //DebugHelper(" Update CrouchLow: " + crouchLow + " CLT: " + crouchLowTime);
+            }
+        }
+
+        private void CheckForCrouchStart(float dv)
+        {
+            if (dv < downThreshold)
+            {
+                crouchStartTime = Time.time;
+            }
+        }
+
+        private void CheckForJumpEnd(float dv)
+        {
+            if (Mathf.Abs(dv) < baselineUpdateSensitivity)
+            {
+                jumping = false;
+                crouchStartTime = 0;
+                //DebugHelper("Ending jump - isFalling = false");
+                // Trick to reset playerPresence so player can jump again
+                playerPresence.StartPhysicsFall(Vector3.zero);
+            }
+        }
+
+        private void UpdateBaseline(float dv)
+        {
+            if (Mathf.Abs(dv) < baselineUpdateSensitivity)
+            {
+                baselineHeight = HMDTransform.position.y;
+                playerPresence.StartPhysicsFall(Vector3.zero);
+            }
+        }
+
+        private float UpdateHeightHistoryAndReturnVelocity(float headHeight)
+        {
+            // TODO - calculation of difference will be off around the y origin
+            var deltaHeight = lastHeight - headHeight;
+            lastHeight = headHeight;
+
+            // update the history and get a sum
+            float heightSum = 0;
+            for (int i = 0; i < historyLen - 1; i++)
+            {
+                heightSum += historicVelocity[i];
+                historicVelocity[i] = historicVelocity[i + 1];
+            }
+            heightSum += historicVelocity[historyLen - 1];
+            historicVelocity[historyLen - 1] = deltaHeight;
+
+            // Velocity, scaled for convenience, signreversed for sanity or lazyness or something
+            return (heightSum / historyLen) * 100 * -1;
+        }
+
+        private void DoJump(float jumpVelocity)
+        {
+            Vector3 forward = GetPlayerForward() * jumpVelocity;
+
+            Vector3 calcVelocity;
+            calcVelocity.x = forwardFactor * forward.x;
+            calcVelocity.y = upFactor * forward.y;
+            calcVelocity.z = forwardFactor * forward.z;
+
+            //DebugHelper("JUMP EVENT: " + calcVelocity.ToString());
+
+            playerPresence.StartPhysicsFall(Vector3.zero);
+
+            playerPresence.GetComponent<Rigidbody>().velocity = calcVelocity;
+        }
+
+        private Vector3 GetPlayerForward()
+        {
+            // necessaries for triangulation
```

Comment:
again, remove this commented out code and just keep things like this in your own dev branch